README.md

```diff
@@ -1,3 +1,33 @@
 # Strapi application
 
-A quick description of your strapi application
+Setting up Strapi
+
+The first thing you do after forking the ACC-repo is to run "npm run build" and "npm run develop" to start the server, but as you will see, its not that easy.
+
+To get started with Strapi you need to install PostgreSQL, and for that you need Homebrew. HomeBrew is a package manager that takes care of everything NPM or Yarn doesn´t. Read more about it on https://brew.sh/
```

Comment:
Unfortunately this is specific to mac, for this part can you make a separate section called "Setting up for mac" and I'll ask essam to create a "Setting up for windows" section

Reply:
Can you make a "Setting up for mac section", right now this implies it works for both operating systems